isolator.py

```diff
@@ -159,98 +162,104 @@ def __init__(
+        )
+        rhou = mass * u
+        energy = rhoe + 0.5 * mass * (u * u)
 
         from pytools.obj_array import make_obj_array
-        from mirgecom.fluid import join_conserved
-        mom = make_obj_array(
-            [
-                0*x_rel
-                for i in range(self._dim)
-            ]
-        )
-        mom[0]=rhou
+        mom = make_obj_array([0 * x_rel for i in range(self._dim)])
+        mom[0] = rhou
+
+        return make_conserved(dim=self._dim, mass=mass, energy=energy, momentum=mom)
 
-        return join_conserved(dim=self._dim, mass=mass, energy=energy,
-                              momentum=mom)
 
 @mpi_entry_point
-def main(ctx_factory=cl.create_some_context,
-         snapshot_pattern="isolator-{step:06d}-{rank:04d}.pkl",
-         restart_step=None, use_profiling=False, use_logmgr=False):
+def main(
+        ctx_factory=cl.create_some_context, rst_filename=None, use_profiling=False,
+        use_logmgr=False, casename="isolator"):
     """Drive the Y0 example."""
+    cl_ctx = ctx_factory()
 
     from mpi4py import MPI
     comm = MPI.COMM_WORLD
-    rank = 0
     rank = comm.Get_rank()
     nparts = comm.Get_size()
 
     """logging and profiling"""
 
-    cl_ctx = ctx_factory()
-    queue = cl.CommandQueue(cl_ctx)
-    actx = PyOpenCLArrayContext(queue,
-        allocator=cl_tools.MemoryPool(cl_tools.ImmediateAllocator(queue)))
-
+    logmgr = initialize_logmgr(use_logmgr,
+        filename=f"{casename}.sqlite", mode="wu", mpi_comm=comm)
 
-    nrestart = 5000
-    nviz = 500
-    current_dt = 1e-7
-    t_final = 1.e0
+    if use_profiling:
+        queue = cl.CommandQueue(cl_ctx,
+            properties=cl.command_queue_properties.PROFILING_ENABLE)
+        actx = PyOpenCLProfilingArrayContext(queue,
+            allocator=cl_tools.MemoryPool(cl_tools.ImmediateAllocator(queue)),
+            logmgr=logmgr)
+    else:
+        queue = cl.CommandQueue(cl_ctx)
+        actx = PyOpenCLArrayContext(queue,
+            allocator=cl_tools.MemoryPool(cl_tools.ImmediateAllocator(queue)))
 
-    dim = 2
-    order = 3
-    exittol = .09
-    current_cfl = 1.0
-    vel_init = np.zeros(shape=(dim,))
-    vel_inflow = np.zeros(shape=(dim,))
-    orig = np.zeros(shape=(dim,))
-    orig[0] = 0.83
-    current_dt = 0.1e-7
+    current_dt = 1e-8
+    t_final = 1e-7
     current_t = 0
-    casename = "isolator"
+    current_step = 0
+    current_cfl = 1.0
     constant_cfl = False
+
     # no internal euler status messages
+    nviz = 500
+    nhealth = 1
+    nrestart = 5000
     nstatus = 25
-    checkpoint_t = current_t
-    current_step = 0
-    so = np.log10(1.e-4/np.power(order,4))
-    epsilon = 3.0e-1*(1.0)/order #This is using a porportionality constant of 3.0e-1 and taking 0p5x as the base grid h
+
+    order = 3
+    so = np.log10(1.0e-4 / np.power(order, 4))
+    epsilon = (
+        3.0e-1 * (1.0) / order
+    )
+    # This uses proportionality constant of 3.0e-1 and taking 0p5x as the base grid h
```

Comment:
Probably put the comment before the line now.
```suggestion
    # This uses proportionality constant of 3.0e-1 and taking 0p5x as the base grid h
    epsilon = (
        3.0e-1 * (1.0) / order
    )
```